### DISCUSSION TITLE
So can this be used to click while a windows is unactive like how ahk used SendControlClick?

### DISCUSSION THREAD

**Discussion by zSynctic** [spyoungtech/ahk#244] [Q&A] [answered]
So can this be used to click while a windows is unactive like how ahk used SendControlClick?

**Comment by spyoungtech** [comment, marked as answer]
Yes. Pretty much all the same capabilities of AHK can be used in this wrapper. The various `Window` methods use `ControlSend` and `ControlClick` by default.

**Reply by zSynctic** [reply]
Can you show me a basic code example which does that since this is the first time for me using this library. thanks

**Reply by spyoungtech** [reply]
Sure. First, make sure that you have everything installed:

```cmd
:: install the package
pip install ahk

:: optionally, install AutoHotkey.exe using pip, if not already installed on your system
pip install "ahk[binary]"
```
Then you can run a script like so:

```python
from ahk import AHK

ahk = AHK()

win = ahk.win_get(title='Untitled - Notepad')
assert win is not None, "Window not found"

# click within in the window
win.click(x=100, y=200)
```

You can also find more examples in the README.

**Reply by zSynctic** [reply]
Thanks. is there a way to change the interval?

**Reply by zSynctic** [reply]
btw for some reason it doesn't work
